pipelines/api/server.py

```diff
@@ -491,7 +491,7 @@ def _get_auth_dict(auth_settings):
         }
 
 
-def make_app(workspace='fixtures/workspace', auth=None):
+def make_app(secret, workspace='fixtures/workspace', auth=None):
```

Comment:
make it a kwargs `cookie_secret=None`

And after, have a 

```
if not cookie_secret:
    raise PipelinesError('Cookie secret is mandatory')
```

Reply:
I thought it should not be optional if the `cookie_secret` is required. So I didn't use `cookie_secret=None` in here.

I checked `None` in https://github.com/Wiredcraft/pipelines/blob/4d6a623896b0f8acdc5f95e3fb46ba74a94c0b08/bin/pipelines#L75